code/addons/interactions/package.json

```diff
@@ -89,7 +89,7 @@
   "devDependencies": {
     "@storybook/jest": "^0.0.10",
     "@storybook/testing-library": "0.0.14-next.0",
-    "@types/node": "^16.0.0 || ^18.0.0",
+    "@types/node": "^16.0.0",
```

Comment:
@valentinpalkovic I needed to do this in order for it to work, can you tell me what is going on?

Reply:
I needed that change because `yarn install` threw an error. Feel free to revert that change, if the error doesn't occur anymore

Reply:
I consulted with @kasperpeulen and we agreed this is better.